src/components/data-management/ProjectCard.jsx

```diff
@@ -0,0 +1,136 @@
+import ProjectDeleteModal from './ProjectDeleteModal';
+import { setActiveProject, updateProject } from '../../redux/actions/projects';
+
+const { Item } = Descriptions;
+
+const validationChecks = [
+  rules.MIN_8_CHARS,
+  rules.MIN_2_SEQUENTIAL_CHARS,
+  rules.ALPHANUM_DASH_SPACE,
+  rules.UNIQUE_NAME_CASE_INSENSITIVE,
+];
+
+const inactiveProjectStyle = {
+  cursor: 'pointer',
+};
+
+const activeProjectStyle = {
+  backgroundColor: blue[0],
+  cursor: 'pointer',
+  border: `2px solid ${blue.primary}`,
+};
+
+const ProjectCard = (props) => {
+  const { projectUuid } = props;
+
+  const dispatch = useDispatch();
+
+  const projects = useSelector((state) => state.projects);
+
+  const [deleteModalVisible, setDeleteModalVisible] = useState(false);
+
+  const { activeProjectUuid } = projects.meta;
+  const project = projects[projectUuid];
+
+  const projectNames = projects.ids.map((uuid) => projects[uuid].name);
+
+  const validationParams = {
+    existingNames: projectNames,
+  };
+
+  const updateProjectName = (newName) => {
+    dispatch(updateProject(project.uuid, { name: newName }));
+  };
+
+  const deleteProject = () => {
+    setDeleteModalVisible(true);
+  };
+
+  return (
+    <>
+      {deleteModalVisible && (
+        <ProjectDeleteModal
+          projectUuid={project.uuid}
+          onCancel={() => { setDeleteModalVisible(false); }}
+          onDelete={() => { setDeleteModalVisible(false); }}
+        />
+      )}
+      <Card
+        data-test-class='project-card'
+        key={projectUuid}
+        type='primary'
+        style={projectUuid === activeProjectUuid ? activeProjectStyle : inactiveProjectStyle}
+        onClick={() => {
+          dispatch(setActiveProject(project.uuid));
+        }}
+      >
+        <Descriptions
+          layout='horizontal'
+          size='small'
+          column={1}
+        >
+          <Item contentStyle={{ fontWeight: 700, fontSize: 16 }}>
+            <EditableField
+              value={project.name}
+              onAfterSubmit={updateProjectName}
+              onDelete={deleteProject}
+              validationFunc={
+                (newName) => validateInputs(
+                  newName,
+                  validationChecks,
+                  validationParams,
+                ).isValid
+              }
+            />
+          </Item>
+          <Item
+            labelStyle={{ fontWeight: 'bold' }}
+            label='Samples'
+          >
+            {project.samples.length}
+
+          </Item>
+          <Item
+            labelStyle={{ fontWeight: 'bold' }}
+            label='Created'
+          >
+            <PrettyTime isoTime={project.createdDate} />
+
+          </Item>
+          <Item
+            labelStyle={{ fontWeight: 'bold' }}
+            label='Modified'
+          >
+            <PrettyTime isoTime={project.lastModified} />
+
+          </Item>
+          <Item
+            labelStyle={{ fontWeight: 'bold' }}
+            label='Last analyzed'
+          >
```

Comment:
instead of saying `labelStyle={{ fontWeight: 'bold' }}` could you not use styled components: https://styled-components.com/ and create a new `BoldItem` component (at the top of this file since it is internal) and use it here instead of `Item`?

Reply:
I've tried using Styled components but the styles aren't transferred to the Item texts. So, I refactored the style out instead.

Reply:
ok, sounds good